app/assets/stylesheets/map/components/registration.sass

```diff
@@ -110,6 +110,12 @@ textarea.sya-registration__textarea
   &::-ms-input-placeholder
     color: $color-grey-light
 
+.error_input
+  border: 1px solid $color-orange !important
+  -webkit-transition: 0.5s
+  transition: 0.5s
+  outline: none !important
```

Comment:
Because our code can be included into other websites (via a javascript embed), we can't pollute the global scope for CSS selectors.

So instead, we should attach this to the selector above it so that it only applies to our CSS classes

```sass
input.sya-registration__input,
textarea.sya-registration__textarea
  // ...

  &.error_input
    border: 1px solid $color-orange !important
    -webkit-transition: 0.5s
    transition: 0.5s
    outline: none !important
```